tests/core/test_parsing.py

```diff
@@ -472,12 +477,16 @@ def test_parse_deceased_field_00(input_, expected):
 
 @pytest.mark.parametrize('filename,expected', [(k, v) for k, v in parse_multiple_scenarios.items()])
 def test_multiple_deceased(filename, expected):
+    """
+    Ensure that the second record yielded by parsing.parse_page
+    is the second deceased person from a collision.
+    """
     page_text = load_test_page(filename)
     content_parser = parsing.parse_page(page_text, 'fake_url')
     _ = next(content_parser)
-    second = next(content_parser)
+    second_person = next(content_parser)
     for key in expected:
-        assert second[key] == expected[key]
+        assert second_person[key] == expected[key]
 
```

Comment:
Same idea here. I don't want to test `parsing.parse_page`, but I want to test the function actually extracting the two fatalities from the deceased field.

Reply:
I didn't intend to get into splitting up the end-to-end tests in this PR. This could be just about improving clarity, to get ready for the structural changes you're planning on introducing in your upcoming PR.